backend/matching-service/README.md

```diff
@@ -0,0 +1,17 @@
+Sample .env
+```
+NODE_ENV=development
+SERVICE_PORT="5200"
+MATCHING_TIMEOUT=10000
```

Comment:
Just curious why you decide to define the timeout threshold as an environmental variable instead of maybe just a constant at where you define the timeout? Any advice will be appreciated, thank you!